### ISSUE TITLE
The Head LR within the DeepLabv3+

### ISSUE THREAD

**Issue by Sun-Rider** [charlesCXK/TorchSemiSeg#7]
Hi,

I notice that you write the comment for the **10 times head learning rate** [here](https://github.com/charlesCXK/TorchSemiSeg/blob/9c569391e99e8bbc6894f2ed65435cc8451e56e6/exp.voc/voc8.res50v3%2B.CPS%2BCutMix/train.py#L134), but seems you do not perform that. Since training the entire network is really time-consuming, may I kindly ask that whether it is the wrong comment or not.

Cheers.

**Comment by charlesCXK** [comment]
Hi, actually it is the wrong comment that comes from an old version of code :)
We will polish the code. Thank you for pointing out!
